tests/Functional/Anonymizer/AnonymizatorTest.php

```diff
@@ -135,7 +135,7 @@ public function testSerial(): void
 
         // Some connectors will return string values for int.
         $actual = \array_map(
-            fn (array $item) => [
+            fn (array $item): array => [
```

Comment:
And here

Reply:
The type is still there

Reply:
Fixed https://github.com/makinacorpus/DbToolsBundle/pull/122/commits/cadcd43377fea26c51e0d8171a40189a7b039b8b